cluster/charts/crossplane/templates/serviceaccount.yaml

```diff
@@ -1,7 +1,8 @@
+{{- if not .Values.serviceAccount.name -}}
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: {{ template "crossplane.name" . }}
+  name: {{ template "crossplane.serviceAccountName" . }}
```

Comment:
Does using `nameOverride` (see discussion: https://github.com/crossplane/crossplane/discussions/3680) not work for your use case?

Reply:
The issue with this is that you are left with a dangling service account that is not used.

In our use case we create and manage all service accounts in a separate project, so being able to provide a SA name is clean and follows other projects way of managing SA (example [Keda](https://github.com/kedacore/charts/blob/main/keda/values.yaml#L127))

Reply:
> In our use case we create and manage all service accounts in a separate project, so being able to provide a SA name is clean and follows other projects way of managing SA (example [Keda](https://github.com/kedacore/charts/blob/main/keda/values.yaml#L127))

@markandersontrocme so is your goal here to allow for _not_ creating an SA rather than just changing its name? If so, this could lead to some surprising behavior. Are you still wanting the [aggregated ClusterRole to be bound](https://github.com/crossplane/crossplane/blob/master/cluster/charts/crossplane/templates/clusterrolebinding.yaml) to your manually managed SA?

Reply:
> is your goal here to allow for not creating an SA
yes

Good point on the ClusterRole, I can update that to point to 

```
subjects:
- kind: ServiceAccount
  name: {{ template "crossplane.serviceAccountName" . }}
```